Code/GraphMol/MolTransforms/MolTransforms.cpp

```diff
@@ -300,6 +300,13 @@ RDGeom::Transform3D *computeCanonicalTransform(const Conformer &conf,
     }
   }
   origin *= -1.0;
+
```

Comment:
I think this is worth a comment.  Also, what happens when two diagonal terms are negative?  This will be true when 1 or 3 are negative.

Reply:
> I think this is worth a comment. 

agreed. I'm also going to re-think the exact logic used

> Also, what happens when two diagonal terms are negative? This will be true when 1 or 3 are negative.

That's a case of two wrongs making a right: since you invert twice, you end up back where you started